client/lib/billing-card/app/models/billing-card-task.js

```diff
@@ -0,0 +1,9 @@
+// import DS   from 'ember-data';
+import Task from 'tahi/models/task';
+
+// BillingCardTask = Task.extend({
+//   qualifiedType: "BillingCard::BillingCardTask"
+// })
+// export default BillingCardTask;
+
+export default Task.extend({});
```

Comment:
Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>Expected '(end)' and instead saw 'default'.